public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -276,20 +276,19 @@ export class TimeSrv {
     this.setAutoRefresh(this.previousAutoRefresh);
   }
 
-  setTime(time: RawTimeRange, fromRouteUpdate?: boolean) {
+  setTime(time: RawTimeRange, updateUrl = true) {
```

Comment:
this inverts the logic here, which means all callers needs to pass false, when they previously passed true